app/models/encryption.rb

```diff
@@ -0,0 +1,10 @@
+class Encryption < ApplicationRecord
+  include PasswordConcern
```

Comment:
@syncrou Should this be EncryptionConcern instead of PasswordConcern?

Reply:
I can change the name.  Jason seemed surprised it wasn't part of `manageiq-password` so it sounds like the plan would be to pull all of it out of our app and move it into the gem.

Reply:
👍